doc/source/ray-core/objects/serialization.rst

```diff
@@ -79,6 +79,57 @@ Serialization notes
 
 - Lock objects are mostly unserializable, because copying a lock is meaningless and could cause serious concurrency problems. You may have to come up with a workaround if your object contains a lock.
 
+Zero-Copy Serialization for Read-Only Tensors
+----------------------------------------------
+Ray provides zero-copy serialization for read-only PyTorch tensors. 
+Ray serializes these tensors by converting them to NumPy arrays and leveraging pickle5's zero-copy buffer sharing. 
+This avoids copying the underlying tensor data, which can improve performance when passing large tensors across tasks or actors.
+
```

Comment:
```suggestion
This avoids copying the underlying tensor data, which can improve performance when passing large tensors across tasks or actors.
However, PyTorch does not natively support read-only tensors, so this feature must be used with caution.

```

Let's move the limitations from below to here after this line instead. Otherwise it's likely that people will miss them (only read to the code sample).